setup.py

```diff
@@ -50,7 +50,7 @@
     'pytest-cov>=1.8.0',
     'pytest-pep8>=1.0.6',
     'pytest>=3.8.0,<5.0.0',
-    'reana-commons[kubernetes]>=0.5.0.dev20190416',
+    'reana-commons[kubernetes]>=0.6.0',
```

Comment:
Beware of possible circular dependencies, in the past we used the following to be safe:

```console
$ git show v0.5.0:setup.py | grep reana-
    'reana-commons>=0.5.0.dev20190416',
    'reana-db>=0.5.0.dev20190416',
```

i.e. the lower dependencies in `pytest-reana` were liberal (since the package is only used by developers) and the lower dependencies in `reana-db` and `reana-commons` were more cleanly delineated:

```console
$ cd ../reana-commons
git show v0.5.0:setup.py | grep pytest-reana
    'pytest-reana==0.5.0',
```

I.e. we can release `pytest-reana` first, using YYYMMDD "dev" dependencies, and then `reana-commons` and `reana-db`, using pytest-reana 0.6.0.

Reply:
Okay, I've updated PR for dev release.

Reply:
Need a DEV minimal version such as `>=0.6.0.dev20191212` then the package would be installable and Travis CI would pass